apps/etterlatte-gyldig-soeknad/src/main/kotlin/no/nav/etterlatte/inntektsjustering/InntektsjusteringRiver.kt

```diff
@@ -52,29 +52,37 @@ internal class InntektsjusteringRiver(
                         return
                     }
 
-            startBehandlingAvInntektsjustering(sak, journalpostResponse, inntektsjustering.id)
+            startBehandlingAvInntektsjustering(sak, journalpostResponse, inntektsjustering)
         } catch (e: JsonMappingException) {
             sikkerLogg.error("Feil under deserialisering", e)
             logger.error("Feil under deserialisering av inntektsjustering (id=${inntektsjustering.id}). Se sikkerlogg for detaljer.")
             throw e
         } catch (e: Exception) {
             logger.error("Uhåndtert feilsituasjon TODO : $", e)
-            throw e
+            // throw e TODO ta stilling til hvordan feil her skal håndteres...
```

Comment:
Er ikke denne tryggere å bare beholde som `throw e` hvis vi ikke har en bedre avklaring?

Reply:
Jeg fjernet den mest fordi det hadde køet opp flere meldinger som gjorde at appen havnet i crash loop. Vi bør gjøre det samme som for søknader. Der kastes ikke feilmelding videre men hvis alt er ok så sendes det en melding tilbake når alt er ok.